Gems/Atom/Feature/Common/Assets/Materials/Types/StandardPBR_LowEndForward.shader

```diff
@@ -7,7 +7,9 @@
 
     "Source" : "./StandardPBR_ForwardPass.azsl",
 
-    "Definitions": [ "QUALITY_LOW_END=1" ],
+    "AddBuildArguments": {
+        "cpp": ["-DQUALITY_LOW_END=1"]
```

Comment:
I actually really dislike this naming convention, given that the source code isn't a `cpp` file and this really common option should be more discoverable.

Reply:
Yeah, it's been bothering me a bit too. We could at least call it "mcpp" since that's the name of the tool we are using for the preprocessing, which would be consistent with the other build argument categories being based on the names of the tools. Or maybe call it "preprocessor"?

Reply:
I was not happy either, but followed the GNU standard, where cpp is the preprocessor, and cxx is c++. Will change to "preprocessor". "mcpp" is tempting but those who come from Objective-C will find it confusing too.

Reply:
I still dislike it :) Can we at least support the top level `Definitions` field in addition? It really is so common of a use case that this is going to bother me forever

Reply:
Done. "Definitions"  are back, at all levels.